ui/src/me/components/ProfileOverlay.tsx

```diff
@@ -0,0 +1,84 @@
+// Libraries
+import React, {PureComponent, ChangeEvent} from 'react'
+import {withRouter, WithRouterProps} from 'react-router'
+import {connect} from 'react-redux'
+
+// Components
+import {Form, Input, Button, ComponentColor} from '@influxdata/clockface'
+import {Overlay} from 'src/clockface'
+
+// Types
+import {AppState} from 'src/types'
+import {ComponentSize, ComponentStatus} from '@influxdata/clockface'
+
+interface StateProps {
+  me: AppState['me']
+}
+
+type Props = StateProps & WithRouterProps
+
+interface State {
+  me: AppState['me']
+}
+
+export class Settings extends PureComponent<Props, State> {
+  constructor(props) {
+    super(props)
+    this.state = {
+      me: this.props.me,
```

Comment:
This is temporary placeholder, right? Until the form is no longer disabled?

Reply:
Yeah basically leaving this here so when the update user route is back this will be an easy piece of work

Reply:
If its really just for a later scenario, remove it. We can pull it back out of history later.